CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.58.0      2024-08-27
+
+* Log Insights: Revamp log parsing mechanism
+  - The new mechanism is more performant and allows for arbitrary
+    log_line_prefix settings. The new parsing mechanism is the default, but you
+    can set `db_log_line_prefix = legacy` in the config file or
+    `LOG_LINE_PREFIX=legacy` in the environment to revert to the old mechanism.
+* Log Insights: Redact parameters from utility statements by default
+  - Statements like `CREATE USER u WITH PASSWORD 'passw0rd'` can leak sensitive
+    data into Log Insights, so they are now redacted by default. Note that these
+    statements are usually very fast, and are normally only logged with Postgres
+    configurations like `log_min_duration_statement = 0` that log every statement.
+* RDS: Update AWS SDK to v1.55.3
+  - This allows using EKS pod identity; documentation coming soon
+* Azure:
+  - Improve log handling for Flexible Server and Cosmos DB
```

Comment:
It may make sense to split them, since the CosmosDB for Postgres support is actually more significant here (it appears that just didn't work in the past).

Maybe:

```suggestion
  - Ensure correct log handling for all Flexible Server events (don't accidentally treat them as Single Server events)
  - Support log parsing for Azure Database for Cosmos DB Postgres
```